ui/app/src/components/MediaCard/MediaCard.tsx

```diff
@@ -106,7 +106,11 @@ function MediaCard(props: MediaCardProps) {
     compact = false
   } = props;
   // endregion
-  const { name, path, type } = item;
+  let { name, path, type } = item;
+  // TODO: search item type comes as 'Other' in audio files, it needs to be updated to 'Audio'
```

Comment:
On the back? Is there a ticket? Start the discussion.

Reply:
Is there anything to update here or is this just back? If just back, remove Todo and follow up on discussion